README.md

```diff
@@ -14,6 +14,8 @@ Can also be installed using
 ext install prettier-vscode
 ```
 
+A word of warning-if you have any other code formatting extensions installed such as for example hugely popular `HookyQR.beautify` or `taichi.react-beautify` they might take precedence and format your code instead of Prettier leading to unexpected results.
```

Comment:
I think that the message itself is good, but it might be a good idea to get the attention of the developer because now it is just a piece of text. If we prepend the message with something like:

```
> ⚠️ A word of warning, if ...
```